### PR TITLE
LVNOD-80 Guest model.

### PR DESCRIPTION
**Pull request by AlbaMontis**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-80)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-81)

## Code reviewers

- [x] @yurii-kokhalevych
- [x] @shyyka
- [x] @yuliannakorol

### Second Level Review

- [x] @lhalam
- [ ] @4asof4ik

### CODE REVIEW

**Review by yurii-kokhalevych — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

File: server/migrations/20170609125500-create-guest.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        allowNull: false,
+        unique: false,
+        references: {
+          model: 'Users',
+          key: 'id'
+        }
+      },
+      status: {
+        type: Sequelize.ENUM,
+        values: ['going', 'notgoing', 'maybe'],
```
Comment (4asof4ik) [inline server/migrations/20170609125500-create-guest.js]: undecided but not maybe

File: server/migrations/20170609125500-create-guest.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        defaultValue: null,
```
Comment (4asof4ik) [inline server/migrations/20170609125500-create-guest.js]: undecided not null

File: server/migrations/20170609125500-create-guest.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        allowNull: true
```
Comment (4asof4ik) [inline server/migrations/20170609125500-create-guest.js]: false

File: server/models/guest.model.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      allowNull: false
+    },
+    user_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      unique: false
+    },
+    status: {
+      type: DataTypes.ENUM,
+      values: ['going', 'notgoing', 'maybe'],
```
Comment (4asof4ik) [inline server/models/guest.model.js]: check migration comment

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Review by yuliannakorol — APPROVED** [review]

**Review by shyyka — APPROVED** [review]

**Review by yurii-kokhalevych — APPROVED** [review]

**Pull request merged**